src/upstream_rbus.c

```diff
@@ -154,16 +154,15 @@ void eventReceiveHandler( rbusHandle_t rbus_Handle, rbusEvent_t const* event, rb
     ParodusPrint("Handling event inside eventReceiveHandler\n");
     (void)rbus_Handle;
     char * interface = NULL;
-    rbusValue_t newValue = rbusObject_GetValue(event->data, "value");
-    rbusValue_t oldValue = rbusObject_GetValue(event->data, "oldValue");
+    rbusValue_t newValue = rbusObject_GetValue(event->data, WEBPA_INTERFACE);
     ParodusInfo("Consumer received ValueChange event for param %s\n", event->name);
-
-    if(newValue) {    
+    if(newValue) {
         interface = (char *) rbusValue_GetString(newValue, NULL);
-	setWebpaInterface(interface);
-    }	
-    if(newValue !=NULL && oldValue!=NULL && interface!=NULL) {
-            ParodusInfo("New Value: %s Old Value: %s New Interface Value: %s\n", rbusValue_GetString(newValue, NULL), rbusValue_GetString(oldValue, NULL), interface);
+        setWebpaInterface(interface);
+        ParodusInfo("New Value : %s, New Interface Value %s\n", rbusValue_GetString(newValue, NULL), interface);
```

Comment:
newValue and interface points to same variable , so can we do getWebpaInterface() and print interface value similar to webcfg change done here, which confirms interface set successfully .

https://github.com/xmidt-org/webcfg/pull/127/files#diff-9e6e5e6a9aed7d99d5bf97b085b55f4d8060290a8ed89ea03ccf6d158c66ee92R1610